.setup/bin/sample_courses/models/course/course_generate_utils.py

```diff
@@ -104,21 +111,29 @@ def make_course_json(self) -> None:
                 max_auto = 0
                 max_ta = 0
 
-                print_grades = True if g_type != 0 or (gradeable.submission_open_date < NOW) else False
-                release_grades = (gradeable.has_release_date is True) and (gradeable.grade_released_date < NOW)
+                if g_type != 0 or (gradeable.submission_open_date < NOW):
+                    print_grades = True
+                else:
+                    print_grades = False
+
+                release_grades = (gradeable.has_release_date is True) and \
+                    (gradeable.grade_released_date < NOW)
 
-                gradeable_config_dir = os.path.join(SUBMITTY_DATA_DIR, "courses", get_current_semester(), "sample",
+                gradeable_config_dir = os.path.join(SUBMITTY_DATA_DIR, "courses",
+                                                    get_current_semester(), "sample",
                                                     "config", "complete_config")
```

Comment:
In my opinion, it's easier to read paths if they are not broken into multiple lines.  You could just suppress the line length warning for paths.

Reply:
Thinking about this again I think I would prefer to just make `os.path.join(. . .)` into a newline instead of ignoring line length. Also, I believe that in flake8 we can ignore warnings on specific lines and pages, but not target paths (unless we get a plugin or make a plugin for flake8 that does that).

Reply:
Talked about this at the Submitty meeting today.  Sounds good to me.